.github/workflows/ci.yml

```diff
@@ -1,4 +1,5 @@
-on: push
+on: 
+  pull_request:
```

Comment:
Note that removing `push` here made it so that now it doens't run CI on `master` branch, only pull requests.